src/common/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -140,9 +140,13 @@ bool ov::pass::MOCTransformations::run_on_model(const std::shared_ptr<ov::Model>
                                                                                 ov::element::u16,
                                                                                 ov::element::i8,
                                                                                 ov::element::u8,
+                                                                                ov::element::u6,
```

Comment:
Minor comment to improve readability: can we use `using namespace ov::element;` and write just precision names (wo namespace) as it is done in src/core/src/preprocess/pre_post_process.cpp?